pilot/pkg/serviceregistry/kube/queue.go

```diff
@@ -93,7 +93,7 @@ func (q *queueImpl) Run(stop <-chan struct{}) {
 	}()
 
 	rate := os.Getenv(enableQueueThrottleEnv)
-	rateLimit := 0
+	rateLimit := 100
```

Comment:
that was the old value ?

Reply:
can you add some logging of which value is used at the end ? just before if rateLimit > 0 on line 105 ?

Reply:
The old value was 10 - but it very insufficient. I can add more logging in separate PR - but the goal is to remove/replace this, so I don't want to spend too much time. You can already see how many events we process per second in the logs.